README.md

```diff
@@ -1,35 +1,6 @@
-[![Build Status](https://semaphoreci.com/api/v1/calico/cni-plugin/branches/master/shields_badge.svg)](https://semaphoreci.com/calico/cni-plugin)
-[![Slack Status](https://slack.projectcalico.org/badge.svg)](https://slack.projectcalico.org)
-[![IRC Channel](https://img.shields.io/badge/irc-%23calico-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#calico)
+## Source code moved
 
-# Calico Networking for CNI
+The Calico source code for this repository has been moved to https://github.com/projectcalico/calico/cni-plugin as part of ongoing efforts to improve
```

Comment:
Maybe we can make the link clickable, slightly nicer for the user so don't get a 404
```
The Calico source code for this repository has been moved to [github.com/projectcalico/calico/cni-plugin](https://github.com/projectcalico/calico/tree/master/cni-plugin) as part of ongoing efforts to improve
```